ground/gcs/src/app/main.cpp

```diff
@@ -264,6 +265,9 @@ int main(int argc, char **argv)
         // https://bugreports.qt-project.org/browse/QTBUG-32789
         QFont::insertSubstitution(".Lucida Grande UI", "Lucida Grande");
     }
+    QProcess process;
```

Comment:
I agree this is ugly and honestly I didn't exhaustively test other approaches working on an OSX vm running on a centrino host is not very fun.
I did try adding LSAppNapIsDisabled key to the dictionary of the plist file (inside gcs/app gets copied to the bundle during build) both as bool and string and it didn't work.